src/lambdas/createUser/createUser.js

```diff
@@ -0,0 +1,68 @@
+const DynamoDB = require('aws-sdk/clients/dynamodb');
+const db = new DynamoDB.DocumentClient();
+const Log = require('@dazn/lambda-powertools-logger');
+const cleanDeep = require('clean-deep');
+const TABLE_NAME = process.env.TABLE_NAME || '';
+
+module.exports.handler = async (event, context) => {
+    try {
+        // get data from event.body
+        if (!event.body) throw 'no body received';
+        const item = typeof event.body === 'object' ? event.body : JSON.parse(event.body);
+
+        // Get UUID for user id from the lambda request id
+        const userId = context.awsRequestId;
```

Comment:
Interesting choice here. Out of curiosity, what is the reasoning for using aws request ID as the UUID of the new user?

Reply:
In the past I've used the 'uuid' package to generate ids.  However, I recently learned that the Lambda request ID is itself a V4 UUID, so I thought it would be good to take advantage of it and save the import.  This is similar, though not exact, to '$util.autoId()' that is available in VTL.